extensions/helix/package.build.ts

```diff
@@ -274,6 +274,230 @@ export const pkg = (modules: Builder.ParsedModule[]) => ({
+                  "if (!workspaceFolder) {",
+                  "  return await fallback()",
+                  "}",
+
+                  "const relativeDirectoryPath = vscode.workspace.asRelativePath(",
+                  "    currentDirectoryUri,",
+                  "    /** includeWorkspaceFolder = */ false",
+                  "  );",
+
+                  "const quickOpenPrefix = relativeDirectoryPath.endsWith('/')",
+                  "  ? relativeDirectoryPath",
+                  "  : relativeDirectoryPath + '/';",
+
+                  "await vscode.commands.executeCommand(",
+                  "  'workbench.action.quickOpen',",
+                  "  quickOpenPrefix,",
+                  ");",
+                ],
+              },
+            ],
+          },
+          "b": {
+            text: "Open buffer picker",
+            command: "workbench.action.showAllEditors",
+          },
+          // "j": {
+          //   text: "Open jumplist picker",
+          //   command: "", // TODO
+          // },
+          "s": {
+            text: "Open symbol picker",
+            command: "workbench.action.gotoSymbol",
+          },
+          "S": {
+            text: "Open workspace symbol picker",
+            command: "workbench.action.showAllSymbols",
+          },
+          "d": {
+            text: "Open diagnostic picker",
+            command: "workbench.action.problems.focus",
+          },
+          "D": {
+            text: "Open diagnostic picker",
+            command: "workbench.action.problems.focus",
+          },
+          "g": {
+            text: "Open changed file picker",
+            command: "workbench.view.scm",
+          },
+          "a": {
+            text: "Perform code action",
+            command: "editor.action.quickFix",
+          },
+          // "'": {
+          //   text: "Open last picker",
+          //   command: "", // TODO
+          // },
+          "G": {
+            text: "Debug",
+            command: "workbench.action.debug.start",
+          },
+          "w": {
+            text: "Window",
+            command: "dance.openMenu",
+            args: [{ menu: "window" }],
+          },
+          "y": {
+            text: "Yank selections to clipboard",
+            command: "dance.selections.saveText",
+            args: [{
+              register: "",
+            }],
+          },
+          "Y": {
+            text: "Yank main selection to clipboard",
+            command: "dance.run",
+            args: [{
+              code: [
+                "const editor = vscode.window.activeTextEditor;",
+                "if (!editor) {",
+                "  return;",
+                "}",
+                "const text = editor.document.getText(editor.selection);",
+                "await vscode.env.clipboard.writeText(text);",
+              ],
+            }],
+          },
+          "p": {
+            text: "Paste clipboard after selections",
+            command: "dance.edit.insert",
```

Comment:
So here I believe this will use the currently selected register, which isn't the case of `Y` (system register). Is that expected?